wagtail/wagtailembeds/embeds.py

```diff
@@ -11,6 +12,7 @@
 
 from wagtail.wagtailembeds.oembed_providers import get_oembed_provider
 from wagtail.wagtailembeds.models import Embed
+from wagtail.utils.deprecation import RemovedInWagtail13Warning
```

Comment:
We should use `RemovedInWagtail14Warning` here instead - we want to give two versions' notice before this is removed.

(`RemovedInWagtail14Warning` has just been added, in 9197bf1259df43a9564495a6b41fb733b025d804)